Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -276,6 +284,8 @@ input[type="password"].remotion-themed,
   background-image: url('../Image/sprite.svg#CheckBoxFalse');
 }
 
+.remotion-themed input[type=checkbox]:not(:disabled):hover + label::before,
+.remotion-themed input[type=checkbox]:not(:disabled):hover + span::before,
```

Comment:
A few lines up we have 
```
.remotion-themed input[type=checkbox] + label:not(:hover)::before,
.remotion-themed input[type=checkbox] + span:not(:hover)::before,
```

I think are should hadd the not-hover rule too.
```
.remotion-themed input[type=checkbox]:not(:hover) + label::before,
.remotion-themed input[type=checkbox]:not(:hover) + span::before,
```
Please check this for the other added rules, too. 

The not-hover rules are added to make it explicit that it's not-hover / the default scenario, so while it's possible that there is no actual effect, having the rules symetrical makes the reasoning easier to follow. OR there is actually a selector-priority-based reason why we need them.

Reply:
Note: already pushed as a new commit to add missing stuff.

Reply:
Comment completed. please review.